### PR TITLE
#164326228 Add JWT implementation on successful login

### PR DESCRIPTION
**Pull request by Benkimeric**
#### What does this PR do?
Implements JWT on login
#### Description of Task to be completed?
-Add JWT 
-Use JWT for users to receive JWT tokens on login
#### How should this be manually tested?

- Clone this repository git clone https://github.com/jkamz/ireporter.git
- Checkout to the branch git checkout ch-164326228-implement-jwt
- Cd into the api directory
- Install all requirements with pip install -r requirements.txt
- Make migrations with the command python manage.py makemigrations
- Migrate the database with python manage.py migrate
- Run server using the command python manage.py runserver
- Go to the URL <your-localhost>/api/auth/login using postman
- Log in using your existing username and password
_ A JWT token will be generated

#### Any background context you want to provide?
None
#### What are the relevant pivotal tracker stories?
[#164326228](https://www.pivotaltracker.com/story/show/164326228)
#### Screenshots (if appropriate)
screenshot from 2019-03-06 07-50-10

#### Questions:
Any suggestions?

### CODE REVIEW

**Comment by AlvinMugambi** [comment]
You left out the relevant PT story on your PR details.
Apart from that and the unittests everything looks fine

**Comment by Benkimeric** [comment]
> You left out the relevant PT story on your PR details.
> Apart from that and the unittests everything looks fine

Thanks for pointing out, I will add it out

**Comment by AlvinMugambi** [comment]
> > You left out the relevant PT story on your PR details.
> > Apart from that and the unittests everything looks fine
> 
> Thanks for pointing out, I will add it out

Add the PT story in the format `[#story-id](story-link)`

**Comment by Benkimeric** [comment]
> I can't see any unit tests for this.

I have added the unit tests

File: api/auth/urls.py [hunk 1]
```diff
@@ -3,12 +3,18 @@
...
+from rest_framework_simplejwt.views import (TokenObtainPairView,
+                                            TokenRefreshView)
+
 
 urlpatterns = [
     path('signup/', SignUpView.as_view(), name='user_signup'),
     path('resend/', ResendActivationView.as_view(), name='user_resend'),
     path('activate/', ActivationView.as_view(), name='user_activate'),
-    path('login/', LoginView.as_view(), name='user_login'),
+    # obtain token pair on login
```
Comment (meshack-mbuvi) [inline api/auth/urls.py]: No need for the comments. Your code should be descriptive enough.

**Comment by Benkimeric** [comment]
This pull request was closed to make changes on branch naming from a Chore `ch` to a Feature `ft`

**Pull request closed without merging**